dev-docs/modules/userId.md

```diff
@@ -186,6 +186,49 @@ gulp build --modules=admixerIdSystem
    });
 {% endhighlight %}
 
+### Adtelligent
+
+The [Adtelligent](https://adtelligent.com) ID system is uniq per-session user identifier for providing high quality DMP data for advertisers
```

Comment:
The [Adtelligent](https://adtelligent.com) ID system is uniq per-session user identifier for providing high quality DMP data for advertisersWrite<br class="Apple-interchange-newline"

should this read:

The [Adtelligent](https://adtelligent.com) ID system is a unique per-session...